README.md

```diff
@@ -9,95 +9,67 @@
 
 ## Install
 
-`adonis install @rocketseat/adonis-bull`
+You need [install Redis Provider](https://preview.adonisjs.com/guides/database/redis/#setup) before install this package.
+
+YARN
 
-## Use
-Register the Bull commands at `start/app.js`
-```js
-const aceProviders = [
-  '@rocketseat/adonis-bull/providers/Command',
-];
+```
+yarn add @rocketseat/adonis-bull
 ```
 
-Register the Bull provider at `start/app.js`
+NPM
 
-```js
-const providers = [
-  //...
-  "@rocketseat/adonis-bull/providers/Bull"
-];
+```
+npm install @rocketseat/adonis-bull
 ```
 
-Create a file with the `jobs` that will be processed at `start/jobs.js`:
+## Use
 
-```js
-module.exports = ["App/Jobs/UserRegisterEmail"];
-```
+Create a file with the `jobs` that will be processed at `start/jobs.ts`:
 
-Add the config file at `config/bull.js`:
+```ts
+const jobs = ["App/Jobs/UserRegisterEmail"]
 
-```js
-"use strict";
-
-const Env = use("Env");
-
-module.exports = {
-  // redis connection
-  connection: Env.get("BULL_CONNECTION", "bull"),
-  bull: {
-    redis: {
-      host: "127.0.0.1",
-      port: 6379,
-      password: null,
-      db: 0,
-      keyPrefix: ""
-    }
-  },
-  remote: "redis://redis.example.com?password=correcthorsebatterystaple"
-};
```

Comment:
What's the replacement for this? How should one configure connections now?

Reply:
This is intentional, now you should define this following the @adonisjs/redis guide.

Reply:
Ah! That makes sense. Thanks for clarifying. Maybe adding a link to Adonis Redis page would be helpful. Currently it links to Bull's config page so I was confused.

Reply:
Done, thanks!